src/tree/updater_quantile_hist.cc

```diff
@@ -35,14 +35,18 @@ namespace tree {
 
 DMLC_REGISTRY_FILE_TAG(updater_quantile_hist);
 
+#if !defined(GTEST_TEST)
```

Comment:
Why?  Is this file included in test?

Reply:
Seems no. It was deleted.

I considered GPU implementation of this parameter as a reference:
https://github.com/dmlc/xgboost/blob/master/src/tree/updater_gpu_hist.cu#L37